calicoctl/calicoctl/commands/convert.go

```diff
@@ -134,23 +140,78 @@ Description:
 }
 
 // convertResource converts v1 resource into a v3 resource.
```

Comment:
nit: probably need to update these comment to also include that it also converts K8s policies

Reply:
sure thing, will do! thanks!